keyboards/wsk/houndstooth/houndstooth.h

```diff
@@ -0,0 +1,22 @@
+#ifndef KB_H
+#define KB_H
+
+#include "quantum.h"
+
+#define LAYOUT( \
+	K00, K01, K02, K03, K04, K05, K40, K41, K42, K43, K44, K45,\
+	K10, K11, K12, K13, K14, K15, K50, K51, K52, K53, K54, K55,\
+	K20, K21, K22, K23, K24, K25, K60, K61, K62, K63, K64, K65,\
+	K30, K31, K32, K33, K34, K35, K70, K71, K72, K73, K74, K75 \
+) { \
```

Comment:
If the layout has to be split up the way it is below, then you should reformat that block like this, so you can use a normal layout, not the split in half thing. 

```suggestion
#define LAYOUT( \
	K00, K01, K02, K03, K04, K05, K20, K21, K22, K23, K24, K25, \
	K40, K41, K42, K43, K44, K45, K60, K61, K62, K63, K64, K65, \
	K10, K11, K12, K13, K14, K15, K30, K31, K32, K33, K34, K35, \
	K50, K51, K52, K53, K54, K55, K70, K71, K72, K73, K74, K75 \
) { \
```